api/openapi-spec/swagger.json

```diff
@@ -13124,6 +13124,18 @@
     "description": "Represents a cloud-init config drive user data source. More info: https://cloudinit.readthedocs.io/en/latest/topics/datasources/configdrive.html",
     "type": "object",
     "properties": {
+     "metaData": {
+      "description": "MetaData contains config drive inline cloud-init metadata.",
+      "type": "string"
+     },
```

Comment:
Instead of making metadata a string, can't you make it a map[string]string instead?